libbeat/docs/outputs/output-logstash.asciidoc

```diff
@@ -0,0 +1,397 @@
+[[logstash-output]]
+=== Configure the Logstash output
+
+++++
+<titleabbrev>Logstash</titleabbrev>
+++++
+
+The Logstash output sends events directly to Logstash by using the lumberjack
+protocol, which runs over TCP. Logstash allows for additional processing and routing of
+generated events.
+
+// tag::shared-logstash-config[]
```

Comment:
I've always wondered why we comment out the tags like this. They work commented and uncommented. Is there a reason we do it this way?

Reply:
Ah crap, I've answered my own question. See comment below.